esp/services/ws_workunits/ws_workunitsHelpers.cpp

```diff
@@ -3502,7 +3490,7 @@ void CWsWuFileHelper::createThorSlaveLogfile(IConstWorkUnit* cwu, WsWuInfo& winf
             {
                 VStringBuffer fileName("%s%c%s_thorslave.%u.%s.log", path, PATHSEPCHAR, processName.str(), i+1, logDate.str());
                 Owned<CGetThorSlaveLogToFileThreadParam> threadParam = new CGetThorSlaveLogToFileThreadParam(
-                    &winfo, nodeGroup, logDate, logDir, i+1, fileName);
+                    &winfo, LINK(nodeGroup), logDate, logDir, i+1, fileName);
```

Comment:
minor: slightly cleaner to not LINK here, but make the member a Linked<INodeGroup> instead of a Owned<INodeGroup>